src/QueryingJson.js

```diff
@@ -8,7 +8,7 @@
   
 
 var paramIsUrlOnly = function(name) {
-  return name.match(/^(json\.nl|json\.wrf|q|wt|start)/);
+  return name.match(/^(json\.nl|json\.wrf|q|wt|start|json2.+)/);
```

Comment:
Same as above:
`<ocd mode>`Would it be better to sort the alternatives? Shouldn't affect the matching, but perhaps looks a bit more orderly, especially if more will need to be added in the future?`</ocd mode>`
```suggestion
  return name.match(/^(json\.nl|json\.wrf|json2.+|q|start|wt)/);
```

Also, my assumption might be wrong, but if we're matching against the parameter name and only the parameter name, would it be better to also match the end of the string, just to be sure it's, e.g., `start` and not `startsomething`, i.e.
```suggestion
  return name.match(/^(json\.nl|json\.wrf|json2.+|q|start|wt)$/);
```

Reply:
what's wrong with json2tsv ? this is parameter for the ambit proxy (not solr) and was working well so far. Thetxt export via proxy will not work if the parameter is removed / changed

Reply:
The problem came that during POST-based request, if a `json2tsv` parameter is passed, it is wrongly assumed to be a parameter for the body, because the `json` prefix is used to treat the parameter as a body one.

The problem showed up, due to export functionality refactoring - merging export and report, and what previously was pure URL-based request, not is a JSON/body based one, and the problem showed up. And, no - this is not Ambit-only situation.

Reply:
ok, understand you are refactoring exports to use POST instead of GET, but 

-  json2tsv is exclusively a parameter used by AMBIT and it is expected to be GET parameter. - Of course we can change the AMBIT server side, but better keep this particular parameter separate, because the JSON body can be passed to SOLR, while json2tsv is handled by the proxy before that . Otherwise the proxy need to parse the JSON body for the sole reason to extract a single parameter.  And of course solr will not understand the json2tsv parameter at all.

Reply:
Ok. I didn't know that, because this is actually a Solr request. But, anyways - `json2tsv` stays as URL parameter - this is exactly the purpose of this fix.

Reply:
OK. Yes, it is a solr request, being proxied for the sole reason of parsing the JSON server side into tab separated spreadsheet - in the refactored system the paring should be implemented client side